contributing.md

```diff
@@ -1,6 +1,7 @@
 ##FAQ
-1. How do I start up the app?
-```meteor --settings settings-development.json```
+
+meteor npm install
```

Comment:
These commands should be in \`\` backticks - like this:  \`meteor npm install\` 
This makes them appear in a monospaced code font, and it helps people reading understand that it's a command.